src/convertToObject.js

```diff
@@ -6,7 +6,25 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const source = sourceString;
+  const splitedByASemicolon = source
+    .split(';')
+    .map((string) => string.trim())
+    .filter((string) => string.length > 0);
```

Comment:
[CODE STYLE] - All chained methods called on the same level should have the same indentation to make the code more readable.